homeassistant/components/logger/services.yaml

```diff
@@ -1,6 +1,25 @@
 set_default_level:
   description: Set the default log level for components.
   fields:
-    level: {description: 'Default severity level. Possible values are notset, debug,
-        info, warn, warning, error, fatal, critical', example: debug}
-set_level: {description: Set log level for components.}
+    level:
+      description: "Default severity level. Possible values are notset, debug, info, warn, warning, error, fatal, critical"
+      example: debug
+
+set_level:
+  description: Set log level for components.
+  fields:
+    homeassistant.core:
```

Comment:
This is not correct. These are all example values. The main difference between this service and other fields is that the examples are not tied to a single field. Not sure what we should do here 🤔

Reply:
I understand I had doubt too. Only one generic line seems not so helpful. So I have decided to use the example of the website documentation.
I'm open for better suggestion.

Reply:
ok, let's keep it.

Reply:
OK cool